packages/appstore/src/components/account/account.tsx

```diff
@@ -1,56 +1,276 @@
+                                sub_account_type={sub_account_type}
+                                has_error={has_error}
+                                platform={platform}
+                                is_dark_mode_on={is_dark_mode_on}
+                                shortcode={shortcode}
+                                should_show_server_name={should_show_server_name}
+                            />
+                        )}
                     </Text>
                     <Text className='account__container--account-details-wrapper--name-number--number'>
-                        {account_number}
+                        {loginid_text}
                     </Text>
                 </div>
 
                 <Text className='account__container--account-details-wrapper--balance'>
-                    {account_balance}
-                    {` ${getCurrencyDisplayCode(currency)}`}
+                    {`${balance} ${getCurrencyDisplayCode(currency)}`}
                 </Text>
             </div>
-            <div className='account__container--account-reset-button'>
-                <Text className='account__container--account-reset-button--label'>
-                    <Localize i18n_default_text={account_button} />
-                </Text>
-            </div>
-            <div className='account__container--dropdown'>
-                <WalletIcon icon={'DropDown'} />
+            <div className='account__container--account-reset-button-wrapper'>
+                {has_reset_balance ? (
+                    <Button
+                        is_disabled={is_disabled}
+                        onClick={(e: React.ChangeEvent<HTMLInputElement>) => {
+                            e.stopPropagation();
+                            onClickResetVirtualBalance();
+                        }}
+                        className='acc-switcher__reset-account-btn'
+                        secondary
+                        small
+                    >
+                        {localize('Reset')}
+                    </Button>
+                ) : (
+                    <Button
+                        is_disabled={is_disabled}
+                        onClick={onClickDeposit}
+                        className='acc-switcher__reset-account-btn'
+                        secondary
+                        small
+                    >
+                        {localize('Deposit')}
+                    </Button>
+                )}
             </div>
+            {isMobile() && getCurrencyDisplayCode(currency) !== 'Demo' && (
+                <div className='account__container--dropdown'>
+                    <WalletIcon icon={'DropDown'} />
+                </div>
+            )}
         </div>
     );
 };
 
 export default OptionsAccount;
+
+const CurrencyDisplay = ({ country_standpoint, currency, loginid, is_virtual }: TCurrentDisplay) => {
+    const user_is_from_this_country_list = Object.values(country_standpoint).includes(true);
```

Comment:
```suggestion
    const is_user_from_list = Object.values(country_standpoint).includes(true);
```

Reply:
Name seems ok I guess since it is more descriptive , if we still need to change better refactor in another PR. since it is not the scope of this PR.

Reply:
you are defining the variable here for the first time, so here is the exact scope for fixing that. for the boolean variables we are using `is` at the beginning of it's name, so one of these could be fine
 ```suggestion
    const is_user_from_country_list
    const is_user_from_list
```

Reply:
this is still not fixed